provider/dynamodb.go

```diff
@@ -414,15 +429,19 @@ func (table dynamodbOnlineTable) Get(entity string) (interface{}, error) {
 	}
 	output_val, err := table.client.GetItem(input)
 	if len(output_val.Item) == 0 {
-		return nil, fferr.NewDatasetNotFoundError(table.key.Feature, table.key.Variant, fmt.Errorf("entity %s not found", entity))
+		wrapped := fferr.NewDatasetNotFoundError(table.key.Feature, table.key.Variant, fmt.Errorf("entity %s not found", entity))
```

Comment:
This should be table not found not entity right?

Reply:
Again, good catch! Thanks for the second set of eyes!